public/js/controllers/home-controller.js

```diff
@@ -120,4 +121,71 @@ function($scope, $http, planService, notificationService) {
             });
         });
     };
+
+    $scope.hasPrereqs = function(course) {
+
+        //This is ugly
```

Comment:
That's very reasonable. However, I dislike the look of the /* above the comment and the misaligned \*/ underneath the comment. So at this point, our disagreement is entirely semantic. You prefer the appearance of one, I prefer the appearance of the other. 

Something like this would need to be decided in a document somewhere that specifies a standard. However, an addition of a standard is an addition of a process in that you now need to maintain that standard. 

In this particular case, the creation of a standard would be false process. We would create a document to mandate commenting styles, and then any developer would need to read the document. Any developer who makes a mistake in following the standard would either 1. be caught in a pull request and need to spend time to fix it or 2. allow code that does not follow the standard to be committed. If 1 happens, we have wasted valuable developer time for them to change something that doesn't actually make us more productive. If 2 happens, we have defeated the purpose of having a standard in the first place, and thus wasted time in creating the process.

So, it's not worth making a standard for something like this. And if there is not a standard for this, there is no need to uphold one.

(If this reads very stern, I apologize. This is good discussion)